socorro/processor/mozilla_transform_rules.py

```diff
@@ -1089,11 +1089,11 @@ def version(self):
 
     #--------------------------------------------------------------------------
     def _predicate(self, raw_crash, raw_dumps, processed_crash, proc_meta):
-        '''addons is a list of tuples containing (extension, version)'''
+        '''addons is a list of string like 'extension:version'. '''
```

Comment:
There's a clash. (sorry for being difficult)
If this doc string is true, the code doesn't need to do `if ':' in addon:`. 
Perhaps the doc string needs to be `addons is a list of string like 'extension:version' but there can be addons that lack the :version part`